modules/pel/peltool/README.md

```diff
@@ -0,0 +1,67 @@
+### Peltool usage
+To know more about peltool capabilities using the help command as shown below
+`Peltool.py --help`
+
+#### List PELs
+- Get list of PELs : `peltool.py -l` (By default serviceable PELs are listed)
+   **Note:** For non-BMC environment user must pass the PEL directory e.g. `peltool.py -p <pel_dir> -l`
+- Get list of PELs including serviceable PELs : `peltool.py -ls`
+- Get list of PELs including non-serviceable PELs: `peltool.py -lN`
+- Get list of PELs including hidden PELs: `peltool.py -lH`
+- Get list of PELs including terminating PELs: `peltool.py -lt`
```

Comment:
I feel we could add example to pass two options together as well.

- Get list of serviceable, non-serviceable, and hidden PELs: `peltool.py -lsNH`